### DISCUSSION TITLE
[Projects-BETA] Unable to navigate back to the organization page

### DISCUSSION THREAD

**Discussion by TheGreatJoules** [community/community#6988] [Projects and Issues]
Not sure if this was called out but when making a new project(beta) unable to navigate back to organization page with an ease of use link, need to either backtrack nth time to the main page.

**Comment by benhutchison** [comment]
Came here to raise the same issue. Please, just a link above the board back to the top level organization for consistent navigation.

**Comment by FlipEverything** [comment]
Me after opening a new project (beta) board:

![image](https://user-images.githubusercontent.com/2407139/141470886-feab5e86-ebb8-4f95-895a-46f4b7fe0966.png)

**Comment by mazsa97** [comment]
+1

**Comment by narumi147** [comment]
+1, a breadcrumb would be nice: `<org>/projects/<project>`

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
